framework_tvb/tvb/interfaces/web/templates/jinja2/visualizers/surface/surface_viewer_controls.html

```diff
@@ -2,8 +2,12 @@
     <dl>
         {% include 'flow/color_scheme.html' %}
         {% with %}
-            {% set urlRegionBoundaries=True %}
+            {% if boundaryURL %}
+                {% set urlRegionBoundaries=True %}
+            {% else %}
+                {% set urlRegionBoundaries=False %}
+            {% endif %}
```

Comment:
In this case you can set directly the urlRegionBoundaries=boundaryURL because it will be checked in the brain_toggle_controls.html file